src/pages/education-resources/index.astro

```diff
@@ -39,5 +39,4 @@ setJumpToState({
   variant="item"
   topic="tutorials"
 >
-  <Content />
```

Comment:
Thanks for the fix! Since we're no longer using this, I think we can delete these lines too, which I believe are only used for creating the `Content` component used here:
https://github.com/processing/p5.js-website/blob/6a83470e7cf64bc30d845f19c8bf01089d559188/src/pages/education-resources/index.astro#L10-L13

Reply:
I removed the three lines.